vector/src/main/java/im/vector/app/features/location/LocationSharingFragment.kt

```diff
@@ -243,7 +242,7 @@ class LocationSharingFragment @Inject constructor(
         // first, update the options view
         val options: Set<LocationSharingOption> = when (state.areTargetAndUserLocationEqual) {
             true  -> {
-                if (BuildConfig.ENABLE_LIVE_LOCATION_SHARING) {
+                if (state.isLiveLocationSharingEnabled) {
```

Comment:
I wouldn't be opposed to injecting the flags directly to the fragment if it helps makes checking them simpler or another option could be to store the `set` in the state instead (and generate it based on the flag)

Reply:
I guess injecting the `VectorFeatures` into the `Fragment` would simplified things. I will go this way.